cluster/common.sh

```diff
@@ -80,7 +80,9 @@ function create-kubeconfig() {
   fi
 
   "${kubectl}" config set-cluster "${CONTEXT}" "${cluster_args[@]}"
-  "${kubectl}" config set-credentials "${CONTEXT}" "${user_args[@]}"
+  if [ -n "${user_args[@]:-}" ]; then
```

Comment:
that's what I discovered in my testing, too, but I didn't see that error in the build log on Jenkins, so I was/am confused.